web/src/components/base/Snackbar.tsx

```diff
@@ -0,0 +1,107 @@
+import CloseIcon from '@mui/icons-material/Close'
+import Alert from '@mui/material/Alert'
+import AlertTitle from '@mui/material/AlertTitle'
+import Button from '@mui/material/Button'
+import IconButton from '@mui/material/IconButton'
+import MUISnackbar from '@mui/material/Snackbar'
+import { styled } from '@mui/material/styles'
+import React, { ReactElement } from 'react'
+import { useTranslation } from 'react-i18next'
+import { Link } from 'react-router-dom'
+
+import { ExtendedSendingStatusType, SNACKBAR_AUTO_HIDE_DURATION } from 'shared'
+
+import useDimensions from '../../hooks/useDimensions'
+
+const StyledMuiSnackbar = styled(MUISnackbar)(({ theme }) => ({
+  [theme.breakpoints.down('lg')]: {
+    marginBottom: 50,
+  },
+}))
+
+const StyledAlert = styled(Alert)`
+  display: flex;
+  align-items: center;
+
+  '& .muialert-action': {
+    margin: 0;
+    padding: 0;
+    align-self: center;
+  }
```

Comment:
It passed the `yarn lint && yarn ts:check` without issues but the IDE says the following:
<img width="462" height="212" alt="Image" src="https://github.com/user-attachments/assets/e252d69f-ab3e-437b-af59-3735430151af" />

🤔  Does it matter if it was small m and a at `.muialert-action`
<img width="923" height="62" alt="Image" src="https://github.com/user-attachments/assets/22908df8-4951-48a2-9fd2-fabab1e3701e" />

Reply:
it was `.MuiAlert-action`, dunno why it is making it small :thinking: